content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,237 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
    Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
    that no driver is selected for those printers.
 
+Epson special case
+~~~~~~~~~~~~~~~~~~
+
+When printing a receipt with an Epson printer, the command that is used is called `GS v 0`. However,
+there are some Epson printers models that do not support this command, they include the following
+models:
+
+- TM-U220
+- TM-U230
+- TM-P60
+- TMP-P60II
+
+If the :abbr:`POS (Point of Sale)` system is using one of these printer models, a particular
+process must be undertaken in order to use another command; `ESC *`.
+
+Process to force ESC * command
+******************************
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `documentation on Epison's site
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for compatible
+   printers.
+
+Should the printer not be compatible with the `GS v 0` command ensure that it is
+compatible with the `ESC *` command.
+
+.. seealso::
+   See the `Epson documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for compatible
+   printers.
+
+If the printer is not compatible with the `ESC *` command then the following process is
+not possible. Should the printer be compatible to use the `ESC *` command to print, follow this
+process to configure the printer with the :abbr:`IoT (Internet of Things)` box.
+
+To manipulate the :abbr:`IoT (Internet of Things)` box to use the `ESC *` command to print, go to
+the :abbr:`IoT (Internet of Things)` box homepage. Access the :abbr:`IoT (Internet of Things)` box
+homepage by entering the :abbr:`IP (Internet Protocol)` address of the :abbr:`IoT (Internet of
+Things)` box into a browser window from a computer on the same network as the :abbr:`IoT (Internet
+of Things)` box (preferably by ethernet connection).
```

Comment:
@tiku-odoo is there not another part of the documentation which already exist to explain how access the homepage that we can refer instead ? I'm afraid that it might confuse people "Ip address" when they can "simply" access it through Odoo IoT app

Reply:
@lse-odoo  

Nice catch! I have made the change to simplify this process to: going to the IOT app --> IoT box --> Click on IP. 

Thanks for suggesting this change.